contents/docs/plugins/build/tutorial.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Tutorial
+sidebar: Docs
+showTitle: true
+---
+
+This tutorial explains the development workflow and best practices, using a hello world plugin. We go from zero to publishing your plugin in the official PostHog repository.
+
+## Pre-Requisites
+
+1. A self-hosted PostHog instance (or a local development environment)
+1. Knowledge of JavaScript (or TypeScript)
```

Comment:
I wonder if we can deliberately lower the bar from "knowledge" to "some knowledge"